backend/routes/api.php

```diff
@@ -199,12 +199,12 @@ function () {
 // Stack route
 Route::prefix("stack")->group(function () {
     Route::post('add', [StackController::class, 'addStack'])->middleware("isloggedin", "isadmin");
-    Route::put('update/{stackId}', [StackController::class, 'updateStack']);
+    Route::put('update/{stackId}', [StackController::class, 'updateStack'])->middleware("isloggedin", "isadmin");
     Route::put('delete/{stackId}', [StackController::class, 'deleteStack']);
```

Comment:
why put when you are deleting in this route?

Reply:
That is the route for edit/update not for delete

Reply:
> That is the route for edit/update not for delete

change the delete route...I commented on the delete roiute not edit route.

Reply:
Ohh. I wasn't the one that worked on the delete route. But i will correct it like you requested